.cypress/utils/event_analytics/constants.js

```diff
@@ -20,6 +20,7 @@ export const TEST_QUERIES = [
   },
   {
     query: 'source = opensearch_dashboards_sample_data_logs',
+    dateRangeDOM: YEAR_TO_DATE_DOM_ID,
```

Comment:
Question: how is this index timestamp determined - is it when you add the data? Will this break next year?

Reply:
timestamp is determined automatically in event explorer by looking for the first encountered date type in an index mappings. It will not break as long as the sample data is not changed.